### PR TITLE
Update Bytes.js binary operations

### PR DESCRIPTION
**Pull request by kartikadur**
operations added: `//, **, %, <<, >>, &, |, ^`
operations updated: `+, /, -`
datatypes included: all except bytearray for +

### CODE REVIEW

**Comment by kartikadur** [comment]
I suspect its probably my local python install misbehaving for some reason, Or it could even be the potatoe I am using for coding. The PR code was passing my local tests. I will adjust the code so that CI's tests pass even if they fail locally here.

**Comment by kartikadur** [comment]
Yay! It worked.

**Comment by kartikadur** [comment]
Hi @glasnt I was wondering if there was anything else needed for this pull request in order to either complete it or discard it as necessary. Let me know so I can do as directed. Thanks :)

File: batavia/types/Bytes.js [hunk 1]
```diff
@@ -326,30 +317,13 @@ Bytes.prototype.__add__ = function(other) {
...
             )
         }
-    } else {
-        throw new exceptions.NotImplementedError.$pyclass('Bytes.__add__ has not been implemented')
+    } else {        
+        throw new exceptions.TypeError.$pyclass("can't concat bytes to " + type_name(other))
     }
 }
 
 Bytes.prototype.__sub__ = function(other) {
```
Comment (brutusthebee) [inline batavia/types/Bytes.js]: At column 13: [(no-trailing-spaces) Trailing spaces not allowed.](http://eslint.org/docs/rules/no-trailing-spaces)

**Comment by kartikadur** [comment]
Thanks

**Pull request merged**